ui/src/app/app.component.html

```diff
@@ -112,7 +112,7 @@
             <fa-icon *ngIf="download.value.status == 'finished'" [icon]="faCheckCircle" style="color: green;"></fa-icon>
             <fa-icon *ngIf="download.value.status == 'error'" [icon]="faTimesCircle" style="color: red;"></fa-icon>
           </div>
-          <span ngbTooltip="{{download.value.msg}}">{{ download.value.title }}</span>
+          <span ngbTooltip="{{download.value.msg}}"><a href="download/{{download.value.filename}}">{{ download.value.title }}</a></span>
```

Comment:
perhaps it's worth it skipping link creation in case the filename property didn't come in (after all its setting is under an "if" in the code in ytdl.py, so it can conceivably not be set in certain circumstances)

Reply:
I will adjust this. However, I am not experienced with Angular, so feel free to comment on the upcoming changes. :)